aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -98,58 +143,128 @@ module AptosFramework::Stake {
         pending_active: vector<ValidatorInfo>,
     }
 
-    /// Any user can delegate a stake.
-    public(friend) fun delegate_stake(account: &signer, to: address, amount: u64, locked_until_secs: u64) acquires StakePool, ValidatorSet {
-        let coins = Coin::withdraw<TestCoin>(account, amount);
+    /// Delegation parameters, set during genesis and stored in @CoreResource account.
+    struct DelegationParameters has key {
+        minimum_delegation: u64,
```

Comment:
I think this may be better a pool parameter instead of a global one. (we'll need to add pool parameters anyway, like commission fees)

Reply:
This should likely be global as we probably don't want to allow validators to set this. They could set it to very low, which would allow spammy delegate_stake calls.